loadtest/user/userentity/scheduled_posts.go

```diff
@@ -0,0 +1,104 @@
+package userentity
+
+import (
+	"context"
+	"fmt"
+	"github.com/mattermost/mattermost/server/public/model"
+)
+
+func (ue *UserEntity) CreateScheduledPost(teamId string, scheduledPost *model.ScheduledPost) error {
+	fmt.Println("CreateScheduledPost: start")
+
+	user, err := ue.getUserFromStore()
+	if err != nil {
+		fmt.Println("CreateScheduledPost: getUserFromStore error", err)
+		return err
+	}
+
+	scheduledPost.UserId = user.Id
+	createdScheduledPost, _, err := ue.client.CreateScheduledPost(context.Background(), scheduledPost)
+	if err != nil {
+		fmt.Println("CreateScheduledPost: CreateScheduledPost error", err)
+		return err
+	}
+
+	id := scheduledPost.ChannelId
+	if createdScheduledPost.RootId != "" {
+		id = scheduledPost.RootId
+	}
```

Comment:
Is this `ChannelId`? Does this mean that we can schedule a single post per channel? This happens in `UpdateScheduledPost` as well.

Reply:
Ah damn, I need to save an array of scheduled posts, not just one scheduled post. Fixed it now.

So yeah, we can have multiple scheduled posts per channel or per thread, this if fixed now.